### PR TITLE
ticking warscore tooltip

### PR DESCRIPTION
**Pull request by Nivaturimika**

### CODE REVIEW

File: src/gui/topbar_subwindows/gui_diplomacy_window.hpp [hunk 1]
```diff
@@ -52,8 +52,7 @@ class cb_wargoal_button : public tinted_button_element_base {
 
 	void button_action(sys::state& state) noexcept override {
 		const dcon::cb_type_id content = retrieve<dcon::cb_type_id>(state, parent);
-		Cyto::Any newpayload = element_selection_wrapper<dcon::cb_type_id>{ content };
```
Comment (schombert) [inline src/gui/topbar_subwindows/gui_diplomacy_window.hpp:55]: you have changed the underlying call from get to send. is that intentional?

Comment (Nivaturimika) [reply src/gui/topbar_subwindows/gui_diplomacy_window.hpp:55]: yeh?

Comment (schombert) [reply src/gui/topbar_subwindows/gui_diplomacy_window.hpp:55]: so the button either wasn't working before or it won't be working now?

Comment (Nivaturimika) [reply src/gui/topbar_subwindows/gui_diplomacy_window.hpp:55]: It's the same, send uses impl get internally, this is just for ease of reading

**Pull request merged**